app/src/main/res/values/themes.xml

```diff
@@ -12,14 +12,28 @@
         <!-- Status bar color. -->
         <item name="android:statusBarColor" tools:targetApi="l">?attr/colorPrimaryVariant</item>
         <!-- Customize your theme here. -->
+        <item name="buttonStyle">@style/Theme.NCNF.BlueButton</item>
+        <item name="android:editTextColor">@color/light_cyan</item>
+        <item name="android:textColorHint">@color/light_cyan</item>
+        <item name="android:windowBackground">@drawable/main_background_gradient</item>
+        <item name="android:textColor">#A7E7FE</item>
```

Comment:
This one too

Reply:
(android:textColor)

Reply:
Same as before, no ?

Reply:
Oh, do you mean the textColor ?

Reply:
As it was originally yes

Reply:
Oh, if you mean as in it's about the same lines as the comment right above, no, the one with value #A7E7FE has to be changed as well

Reply:
> Oh, do you mean the textColor ?

Yes

Reply:
Done